Dockerfile.local

```diff
@@ -0,0 +1,27 @@
+FROM fedora:24
+ENV LANG C
+
+RUN echo "deltarpm=0" >> /etc/dnf/dnf.conf
+RUN dnf -y update
+# behave: core
+# httpd: old-style repos
+# six: py2/py3
+# enum34: rpmdb State
+# whichcraft: shutil.which() for py2
+# jinja2: rpmspec template
+RUN dnf -y install dnf-plugins-core python3-dnf-plugins-core python2-dnf-plugins-core httpd python2-behave python2-six python-enum34 https://kojipkgs.fedoraproject.org//packages/python-whichcraft/0.4.0/2.fc24/noarch/python2-whichcraft-0.4.0-2.fc24.noarch.rpm python-jinja2
```

Comment:
I think here should go only test requirements and then we should install all packages from nightlies.